tests/test_route.py

```diff
@@ -112,14 +98,12 @@ def test_RouteAddRemoveIpv4Route(self, dvs, testlog):
         time.sleep(1)
 
         # check application database
-        tbl = swsscommon.Table(self.pdb, "ROUTE_TABLE")
-        route_entries = tbl.getKeys()
+        route_entries = self.pdb.get_keys("ROUTE_TABLE")
         assert "2.2.2.0/24" in route_entries
```

Comment:
There's a method here that you can use to guarantee that this key exists in the route table before proceeding: https://github.com/Azure/sonic-swss/blob/3052fb5f25fea0ba2f60ad17c294e6da8b9eb70a/tests/dvslib/dvs_database.py#L326

Reply:
Since this line only checks for one entry, I am using wait_for_entry() instead.